sway-core/src/type_engine.rs

```diff
@@ -75,6 +75,11 @@ impl ToJsonAbi for TypeId {
                     .map(|x| x.generate_json_abi())
                     .collect(),
             ),
+            TypeInfo::Array(type_id, _) => Some(vec![Property {
+                name: "r#array_element".to_string(),
```

Comment:
I didn't know what name to choose specifically, but I guess it doesn't matter. Also, there should only ever be a single entry in this vector of `Property`s.